pkg/middleware/logging/logging.go

```diff
@@ -143,6 +154,8 @@ func reportable(l logger.Logger) interceptors.CommonReportableFunc {
 
 		return &reporter{
 			ctx:            ctxzap.ToContext(ctx, zapLogger.Logger),
+			grpcCtx:        ctx,
```

Comment:
Why do you need a new `ctx` object? The one above it works

Reply:
This is unresolved

Reply:
Thank you for your patience.  😅